lutris/runners/steam.py

```diff
@@ -166,7 +176,10 @@ def steam_data_dir(self):
                 return path.rstrip('sSteamAp')
 
     def get_executable(self):
-        return system.find_executable('steam')
+        if system.find_executable('lsi-steam') and self.runner_config.get('lsi_steam'):
```

Comment:
Put the config condition first so we don't do a lookup for the executable if the config isn't set